docs/oneliners.rst

```diff
@@ -0,0 +1,347 @@
+.. -*- mode: rst -*-
+
+Oneliners
+=========
+
+Yellowbrick's quick methods are visualizers in a single line of code!
+
+Yellowbrick is designed to give you as much control as you would like over the plots you create, offering parameters to help you customize everything from color, size, and title to preferred evaluation or correlation measure, optional bestfit lines or histograms, and cross validation techniques. To learn more about how to customize your visualizations using those parameters, check out the :doc:`api/index`.
+
+But... sometimes you just want to build a plot with a single line of code!
+
+On this page we'll explore the Yellowbrick quick methods (aka "oneliners"), which return a fully fitted, finalized visualizer object in only a single line.
```

Comment:
Do you think adding something like: 

> "The visualizer gives you access both to the drawing and machine learning contexts. To modify the figure you can access `viz.fig` and `viz.ax`. It also gives you access to the underlying estimator and learned attributes such as ``score_``, ``class_counts_``, etc." 

Would help or just confuse things?

Reply:
I really like this idea, but I'm thinking this might be a good fit for the FAQ &mdash; I can add in a section about quick methods when I do my next little PR updating our FAQ with info on using YB with fitted/unfitted models (and I'll make sure there's a link to that FAQ from the oneliners page)!